src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -78,12 +79,17 @@ public final class Utilities {
     public static final String ISO_8601_TIMEDATE_FORMAT_WITH_MS = "yyyy-MM-dd'T'HH:mm:ss.SSSX";
 
     /**
-     * CCS directory where data files are stored (under problem short name).
+     * CLICS directory where judge's (secret) data file are stored
      * 
      * @see #getSecretDataPath(String, Problem)
      * @see #getSecretDataPath(String, String)
      */
-    public static final String SECRET_DATA_DIR = "data" + File.separator + "secret";
+    public static final String SECRET_DATA_DIR = "data" + File.separator + ExportYAML.SECRET_DIRECTORY_NAME;
```

Comment:
It is very confusing that there are two different definitions in the code for the "secret data directory name".  Here, it defines `SECRET_DATA_DIR`, which sounds for all the world like _the name of the directory which holds the secret data_.  However, class `ExportYAML` _also_ contains what sounds like an equivalent definition:  `SECRET_DIRECTORY_NAME`.  It seems like there should be something to more clearly distinguish between these two.  Perhaps the constant defined here (in `Utilities`) should really be something like `SECRET_DATA_DIR_PATH`, since it's defining a _path_ rather than a specific directory name?

It also bothers me that the definition of `SECRET_DIRECTORY_NAME` is contained in class `ExportYAML`.  Use of the "secret directory name" is clearly not just something relating to YAML exporting.  I think that constant belongs elsewhere (maybe in `Constants.java`?)